docs/learn/learn-opengov.md

```diff
@@ -13,323 +13,335 @@ import VLTable from "./../../components/Voluntary-Locking";
+  applied
+- **Approval Curve**: the curve describing the minimum % of _aye_ votes as a function of time within
+  the Decision Period. The approval % is the portion of _aye_ votes (adjusted for conviction) over
+  the total votes (_aye_, _nay_, and _abstained_).
+- **Support Curve**: the curve describing the minimum % of all votes as a function of time within
+  the Decision Period. The support % is defined as the portion of all votes without conviction (i.e.
+  _aye_, _nay_ and _abstained_) over the total possible amount of votes in the system. Support is a
+  measure of turnout.
+
+For example, a runtime upgrade (requiring a `set_code` call, if approved) does not have the same
+implications for the ecosystem as the approval of a treasury tip (`reportAwesome` call), and
+therefore different Origins for these two actions are needed in which different deposits, turnouts
+(i.e. support), approvals, and a minimum [enactment](#enactment) periods will be predetermined on
+the pallet.
+
+For detailed information about origin and tracks, and parameter values in Kusama, see
+[this page](../maintain/maintain-guides-opengov.md#origins-and-tracks-info).
+
+### Approval and Support
+
+:::info Adaptive Quorum Biasing is deprecated
+
+In OpenGov, [Adaptive quorum biasing](./learn-governance.md#adaptive-quorum-biasing) used in
+Governance V1 has been replaced with the **Approval and Support system**.
 
-{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} utilizes a concept called
-`Voluntary Locking` which allows token holders to increase their voting power by declaring how long
-they are willing to lock up their tokens, hence, the number of votes for each token holder will be
-calculated using the following formula:
+:::
 
-```
-votes = tokens * conviction_multiplier
-```
+![opengov-approval-support](../assets/opengov-approval-support.png)
+
+The figure above provides a summary view of how the approval and support system works during the
+Decision Period.
+
+Once the proposal exits the Lead-in Period and enters the Voting Period, to be approved, it must
+satisfy the approval and support criteria for the minimum **Confirmation Period**.
+
+- **Approval** is defined as the share of approval (_aye_ votes) vote-weight (after adjustment for
+  [conviction](#voluntary-locking)) against the total vote-weight (for all approval, rejection, and
+  abstained).
+- **Support** is the total number of votes (ignoring any adjustment for conviction) compared to the
+  total possible votes that could be made in the system.
+
+The figure shows that even if the approval criteria are satisfied (i.e. % approval is greater than
+the approval curve), the proposal only enters the confirmation period once the support criteria are
+satisfied (i.e. % support is greater than the support curve). If the referendum meets the criteria
+for at least the minimum confirmation period, then the proposal is approved and scheduled for
```

Comment:
There is no minimum here. It is just the confirmation period of the track, which is a predefined value.

```suggestion
for the confirmation period, then the proposal is approved and scheduled for
```